src/Host.cpp

```diff
@@ -241,7 +241,7 @@ void Host::saveModules(int sync, bool backup)
             QString packagePathName = mudlet::getMudletPath(mudlet::profilePackagePath, mHostName, moduleName);
             filename_xml = mudlet::getMudletPath(mudlet::profilePackagePathFileName, mHostName, moduleName);
             int err;
-            zipFile = zip_open(entry[0].toStdString().c_str(), 0, &err);
+            zipFile = zip_open(entry[0].toStdString().c_str(), ZIP_CREATE, &err);
```

Comment:
`zip_open` was added to `libzip` version 1.0 - and I notice that there does not appear to be any version checking code to handle *admittedly obsolete now* 0.1x versions - do we have a precise minimum libzip version number on record anywhere - and, if so, what is it {remembering that the last working libzip that can be compiled **without** *CMake* is 1.3.0 - which has relevance for *qmake* powered builds on Windows}?

N.B. Should this command fail I do not see any checks being made for `zipfile` being a `NULL` (it is a C library so does `nullptr` still work in them?).

Also, I think `toStdString().c_str()` uses an implicit UTF-8 conversion - does this work on non-ASCII path/filenames on Windows builds? I guess we'll find out eventually...